### PR TITLE
add cloudflare ip ranges

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

File: _docs/_api/network_connectivity_issues.md [hunk 1]
```diff
@@ -26,10 +26,11 @@ For more information about MTR, check out these resources based on your operatin
 
 ## Allowlisting Braze API endpoints IP ranges
 
-To allowlist Braze API endpoints through your firewall, our CDN provides access to the list of assigned IP ranges via a JSON dump. For a list of Fastly.com IP ranges, refer to their [public IP list][3].
+To allowlist Braze API endpoints through your firewall, our CDN provides access to the list of assigned IP ranges via a JSON dump. For a list of Braze API IP ranges, refer to both [Fastly's public IP list][3] and [Cloudflare's public IP list][6].  Note that these IPs may change at any time.
```
Comment (josh-mccrowell-braze) [inline _docs/_api/network_connectivity_issues.md:29]:
```suggestion
To allowlist Braze API endpoints through your firewall, our CDN provides access to the list of assigned IP ranges via a JSON dump. For a list of Braze API IP ranges, refer to both [Fastly's public IP list][3] and [Cloudflare's public IP list][6]. Note that these IPs may change.
```

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Comment by josh-mccrowell-braze** [comment]
Hi @evin-bz : Thanks for drafting this up! The French docs will be handled downstream from us, so we can go ahead and action this in the English version. 

The only blocker right now is the fact that we require contributors to sign our [Contributor License Agreement](https://www.braze.com/docs/cla). If you would please sign this and ping me, I'll make sure that it's merged in. 

Thank you!

**Comment by evin-bz** [comment]
Done! Thanks for the help @josh-mccrowell-braze

**Comment by josh-mccrowell-braze** [comment]
@cla-bot check

**Review by rachel-feinberg — APPROVED** [review]
lgtm

**Pull request merged**